src/test/java/eu/cessda/pasc/oci/ConsumerSchedulerTest.java

```diff
@@ -48,7 +48,10 @@
  */
 public class ConsumerSchedulerTest {
     // mock for debug logging
-    private final AppConfigurationProperties appConfigurationProperties = mock(AppConfigurationProperties.class);
+    private final AppConfigurationProperties appConfigurationProperties = new AppConfigurationProperties(
+            null, List.of("cs", "da", "de", "el", "en", "et", "fi", "fr", "hu", "it", "nl", "no", "pt", "sk", "sl", "sr", "sv"),
```

Comment:
Dos this mean that the list of languages has to be maintained in 2 places? 
See also AppConfigurationProperties.java

Reply:
Not in this case, as this is test code that initialises `AppConfigurationProperties` with a known set of languages. This is so tests aren't affected by the addition and removal of language support.